router/parser/qparser.go

```diff
@@ -211,7 +209,7 @@ func (qp *QParser) Parse(query string) (ParseState, string, error) {
 		case lyx.VarTypeSet, "":
 			varStmt := ParseStateSetStmt{}
 			varStmt.Name = q.Name
-			if len(varStmt.Value) > 0 {
+			if len(q.Value) > 0 {
```

Comment:
Why?

Reply:
Because we init varStmt in this block without any parameters. And `len(varStmt)` will be equal 0 at any time